CHANGES.txt

```diff
@@ -21,6 +21,10 @@ Changes
 Fixes
 =====
 
+ - Do not include ``BLOB`` tables in snapshot when using ``CREATE SNAPSHOT``
+   with ``ALL`` keyword, because ``BLOB`` tables cannot be backed up using
+   ``SNAPSHOT/RESTORE`` functionality.
```

Comment:
hm this reads a bit contradictorily. maybe rephrase to: "Do not include lucene indices(metadata) of ``BLOB`` tables .... because binary blobs of ``BLOB`` tables cannot be..." a bit long and maybe to detailed but imho more understandable. maybe be you'll have a better suggestion ;)